autoreact/ktp_xarray/xarray_wrappers.py

```diff
@@ -18,75 +17,106 @@ def from_data(temps, press, rates):
 
 # Getters
 def get_pressures(ktp):
-    """
-    Gets the pressure values
-    """
+    """Gets the pressure values"""
 
     return ktp.pres.data
 
 
 def get_temperatures(ktp):
-    """
-    Gets the temperature values
-    """
+    """Gets the temperature values"""
 
     return ktp.temp.data
 
 
 def get_values(ktp):
-    """
-    Gets the KTP values
-    """
+    """Gets the KTP values"""
 
     return ktp.values
 
 
 def get_pslice(ktp, ip):
-    """
-    Get a slice at a selected pressure value
-    """
+    """Get a slice at a selected pressure value"""
 
     return ktp.sel(pres=ip)
 
 
 def get_tslice(ktp, it):
-    """
-    Get a slice at a selected temperature value
-    """
+    """Get a slice at a selected temperature value"""
 
     return ktp.sel(temp=it)
 
 
 def get_spec_vals(ktp, it, ip):
-    """
-    Get a specific value at a selected temperature and pressure value
-    """
+    """Get a specific value at a selected temperature and pressure value"""
 
     return ktp.sel(temp=it, pres=ip)
 
 
 def get_ipslice(ktp, ip):
-    """
-    Get a slice at a selected pressure index
-    """
+    """Get a slice at a selected pressure index"""
 
     return ktp.isel(pres=ip)
 
 
 def get_itslice(ktp, it):
-    """
-    Get a slice at a selected temperature index
-    """
+    """Get a slice at a selected temperature index"""
 
     return ktp.isel(temp=it)
 
 
 
 # Setters
 def set_rates(ktp, rates, pres, temp):
-    """
-    Sets the KTP values
-    """
+    """Sets the KTP values"""
 
     ktp.loc[{"pres": pres, "temp": temp}] = rates
     return ktp
```

Comment:
I would pass this back through your `from_data` function above before returning:
```
    return from_data(
        temps=temperatures(ktp),  # currently called `get_temperatures()`
        press=pressures(ktp), # currently called `get_pressures()`
        rates=values(ktp) # currently called `get_values()`
    )
```
I have found it very convenient to use `from_data` as a central function for validating and normalizing data, so that, if I ever need to make a change to those things, the change is concentrated in one function rather than distributed across the code.